views/lib/extended_footer.hbs

```diff
@@ -19,12 +19,12 @@
 							rel="noopener">{{$t "lib.global.link.termsOfUseHPI" }}</a>
 					</li>
 					<li><a href="{{@root.theme.documents.specificFiles.privacyExemplary}}" target="_blank"
-							rel="noopener" aria-label="{{$t "lib.login.headline.onlyPrivacy"}} PDF {{$t "global.openInNewTabWarning"}}">{{$t "lib.global.link.dataProtectionDeclaration" }}</a>
+							rel="noopener">{{$t "lib.global.link.dataProtectionDeclaration" }}</a>
 					</li>
 					<li><a href="/team">{{$t "lib.global.link.team"}}</a></li>
-					<li><a href="/partner">{{$t "lib.global.link.partner"}}</a></li>
-					<li><a href="https://github.com/hpi-schul-cloud/" target="_blank" rel="noopener" aria-label="{{$t "lib.global.link.github"}} {{$t "global.openInNewTabWarning"}}">{{$t "lib.global.link.github"}}</a></li>
```

Comment:
Is there a reason for deleting all the aria-labels here?

Reply:
recovered them ;)

Reply:
a couple other aria-labels were still removed in this file... what's up with them?